static/js/emoji_picker.js

```diff
@@ -232,15 +232,28 @@ function filter_emojis() {
                 continue;
             }
             const emojis = category.emojis;
-
             for (const emoji_dict of emojis) {
+                let match = false;
                 for (const alias of emoji_dict.aliases) {
-                    const match = search_terms.every((search_term) => alias.includes(search_term));
+                    match = search_terms.every((search_term) => alias.includes(search_term));
                     if (match) {
                         search_results.push({...emoji_dict, emoji_name: alias});
                         break; // We only need the first matching alias per emoji.
                     }
                 }
+
+                if (!match && emoji_dict.emoji_code) {
```

Comment:
We should check that it's a unicode emoji before inspecting `emoji_code` here.

Reply:
thank you for the hint, I have updated the PR to check for `reaction_type`

Reply:
Hmm, so I've just noticed that the `reaction_type` attribute is not always available on the emoji objects in the emoji picker, however the `is_realm_emoji` boolean is always available. Would it suffice to do a check on that instead? 🤔 
By breezing through, I'm of the impression that `is_realm_emoji===false` means `unicode`